stl/inc/functional

```diff
@@ -1614,7 +1607,16 @@ public:
 // FUNCTION TEMPLATE bind_front
 template <class _Fx, class... _Types>
 _NODISCARD constexpr auto bind_front(_Fx&& _Func, _Types&&... _Args) {
-    return _Front_binder<_Fx, _Types...>(_STD forward<_Fx>(_Func), _STD forward<_Types>(_Args)...);
+    static_assert(is_constructible_v<decay_t<_Fx>, _Fx>,
+        "std::bind_front requires the decayed callable to be constructible from an undecayed callable");

```

Comment:
As far as I can tell `std::` is conventionally omitted in error messages

Reply:
I think we're inconsistent: `"[^"]*std::` has 152 hits, the vast majority of which are `static_assert` messages.

Reply:
While I am the top proponent of omitting `std` in bug report titles, test code, etc., I believe that `static_assert` messages (and similar diagnostics like deprecation warnings) are a case where `std::` provides value. This is because there are lots of similarly-named types (e.g. `boost::shared_ptr`, `gsl::span`, `absl::string_view`). The filename and (hopefully) template instantiation context provide info, but mentioning the namespace in the message seems like a good way to avoid confusion.